demisto_sdk/commands/generate_docs/generate_playbook_doc.py

```diff
@@ -127,6 +127,8 @@ def get_playbook_dependencies(playbook: dict, playbook_path: str) -> Tuple[list,
                                 integrations.add(integration_dependency)
         else:
             script_name = task.get('scriptName')
+            if not script_name:
```

Comment:
you can do this change in one line - might look better

Reply:
done.